services/brew-registry-pullsecret/admin_rbac.yaml

```diff
@@ -27,3 +27,32 @@ subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
   name: brew-registry-pullsecret-admins
+---
+- kind: ClusterRole
```

Comment:
1. This is invalid manifest YAML input, this is not supposed to be a list. The file is a series of YAML documents (separated by `---`), each document is a single YAML manifest. You added a list of two individual manifests, which is not a valid manifest itself (you would need a `List` manifest for that.

2. This RBAC will not solve the issue: `secrets "registry-pullsecret" is forbidden: User "system:serviceaccount:ci:ci-operator" cannot get secrets in the namespace "brew-registry-pullsecret"`. You need to allow that SA to get secrets in that namespace, not manipulate Prowjobs.

Reply:
The clusterrole.rbac.authorization.k8s.io "prowjob-access" not found part is just misleading :(